PyPowerStore/powerstore_conn.py

```diff
@@ -67,3 +68,4 @@ def __init__(
         self.file_dns = FileDNS(self.provisioning, enable_log=enable_log)
         self.file_nis = FileNIS(self.provisioning, enable_log=enable_log)
         self.snmp_server = SNMPServer(self.provisioning, enable_log=enable_log)
+        self.metrics= MetricsFunctions(self.provisioning, enable_log=enable_log)
```

Comment:
Please add a new line
```suggestion
        self.metrics= MetricsFunctions(self.provisioning, enable_log=enable_log)

```